### ISSUE TITLE
Position relative

### ISSUE THREAD

**Issue by elverskog** [cmswalker/fullpage-react#39]
A small note. I needed to place wrapper around each slide, with `position: relative`, to allow any absolute positioned items (like a down-arrow button), inside my slide, to move along with the slide.

          <Slide>
            <div style={styles.slide}>
              <HomeWhat {...props} />
            </div>
          </Slide>

Any chance  you may want to add position relative to each slide? It doesn't seem to cause any negatives.

**Comment by cmswalker** [comment]
@elverskog I'm actually almost done with v3, it will have some small breaking changes. Instead of getting access to the Slide component you will just pass slides as props, it will address this change along with lots of scroll improvements. I will double check this test case and add it if it isn't already fixed in the changes I've made, but I'm pretty sure I've already handled this. Will ping you when it's released.
